tgui/packages/tgui/interfaces/CloningPod.js

```diff
@@ -0,0 +1,104 @@
+import { useBackend } from '../backend';
+import { Button, ProgressBar, Section, Box, Flex, Icon, NumberInput } from '../components';
+import { Window } from '../layouts';
+
+export const CloningPod = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    biomass,
+    biomass_storage_capacity,
+    sanguine_reagent,
+    osseous_reagent,
+    organs,
+    currently_cloning
+  } = data;
+  return (
+    <Window resizable>
+      <Window.Content scrollable>
+        <Section title="Liquid Storage">
+          <Flex height='25px' align='center'>
```

Comment:
I think you want a labelled list here

Reply:
I'm using a `Flex` element because Sanguine Reagent and Osseous Reagent both have more than one input assoc. with them